Add renovate.json configuration (#205)
devfile/registry: CONTRIBUTING.md

  
    - Make sure the name matches the devfile stack's name and be in the format `<language>-<framework>`. E.g. `java-quarkus`, `python-django`, etc.

> 5) In case one of your components has an image, use a fixed version tag (e.g `<image>:1.1.0`) instead of `latest`. Our renovate bot will take care of the updates to your image's tag.

Reviewer: ```suggestion
5) In case one of your components has a reference to an image, use a fixed version tag (e.g `<image>:1.1.0`) instead of `latest`. Our renovate bot will take care of the updates to the image tags.
```

    - Minimally, testing with odo v2 (`odo create`, `odo push`, etc) and odo v3 (`odo init`, `odo dev`, etc) is recommended, however if your Devfile is used with other tools, it's recommended to test there as well.
5) Open a pull request against this repository with your changes.

### Automatic Stack Image Update

> As images used inside the stacks need to be up-to-date and in order to avoid using the `latest` tag, the renovate bot runs periodically ensuring that all images used from stacks (for example, inside components) have the latest version. As a result, all images used inside a devfile of a stack need to have a fixed version (e.g `1.1.0`) instead of `latest`.

Reviewer: ```suggestion
Avoid using the `latest` tag, the renovate bot runs periodically ensuring that all image references used from stacks (for example, inside components) have the latest version. As a result, all image references used inside the stack devfiles will need to have a fixed version (e.g `1.1.0`) instead of `latest`.
```
